chia/data_layer/data_store.py

```diff
@@ -369,8 +369,15 @@ async def _insert_terminal_node(self, key: bytes, value: bytes) -> bytes32:
     async def get_pending_root(self, tree_id: bytes32) -> Optional[Root]:
         async with self.db_wrapper.reader() as reader:
             cursor = await reader.execute(
-                "SELECT * FROM root WHERE tree_id == :tree_id AND status == :status LIMIT 2",
-                {"tree_id": tree_id, "status": Status.PENDING.value},
+                """
+                SELECT * FROM root WHERE tree_id == :tree_id
+                AND status IN (:pending_status, :pending_batch_status) LIMIT 2
```

Comment:
I'm a bit iffy about combining 'waiting for burying on chain' and 'in the middle of dealing with practical limits of inserting' as both being 'pending', particularly given the historical usage here.